sim/src/main/makefrag/midasexamples/Makefrag

```diff
@@ -62,9 +62,10 @@ driver_dir = $(firesim_base_dir)/src/main/cc
 firesim_lib_dir = $(firesim_base_dir)/firesim-lib/src/main/cc/
 DRIVER_H = $(shell find $(driver_dir) -name "*.h")
 DRIVER_CC := $(driver_dir)/midasexamples/Driver.cc \
-             $(wildcard $(addprefix $(firesim_lib_dir)/, $(addsuffix .cc, bridges/*)))  \
+			$(wildcard $(addprefix $(firesim_lib_dir)/, $(addsuffix .cc, bridges/* )))  \
+			$(RISCV)/lib/libfesvr.a \
 
-TARGET_CXX_FLAGS := -DDESIGNDRIVERCLASS=$(DESIGN)_t -DDESIGNNAME_$(DESIGN) -I$(driver_dir) -I$(firesim_lib_dir) -I$(driver_dir)/midasexamples -g
+TARGET_CXX_FLAGS := -L$(RISCV)/lib -DDESIGNDRIVERCLASS=$(DESIGN)_t -DDESIGNNAME_$(DESIGN) -I$(driver_dir) -I$(RISCV)/include -I$(driver_dir)/midasexamples -g
```

Comment:
You never seem to reference the library with `-lfesvr`? Am I missing something here?

Reply:
`$(RISCV)/lib/libfesvr.a` is included in the sources list.  @davidbiancolin , I'm assuming that you had linker problems if you didn't also include `-L$(RISCV)/lib`?

Reply:
I did. But now i'm second guessing myself. I'll double check